### PR TITLE
Bump phpstan to level 5

### LINKED ISSUE #201: Bump phpstan to level 5
### Improvement description

.

### PR DESCRIPTION
**Pull request by robertSt7**
Resolves #201

### CODE REVIEW

File: src/Processing/ImportProcessingService.php [hunk 1]
```diff
@@ -113,8 +113,10 @@ public function __construct(QueueService $queueService, MappingConfigurationFact
         $this->configLoader = new ConfigurationPreparationService();
     }
 
-    public function processQueueItem(int $id)
+    public function processQueueItem(string $id)
```
Comment (fashxp) [inline src/Processing/ImportProcessingService.php]: why did you change that?

Comment (fashxp) [reply src/Processing/ImportProcessingService.php]: I would say we should do the casting outside if necessary ...

**Review by fashxp — CHANGES_REQUESTED** [review]

File: src/Processing/ImportProcessingService.php (hunk 1 above)
Comment (robertSt7) [reply src/Processing/ImportProcessingService.php]: I totally aggree with you. I will change that.

**Review by fashxp — APPROVED** [review]
